reference/manual-upgrade.md

```diff
@@ -0,0 +1,391 @@
+    {% tabs %}
+    {% tab title="Command to check the helm list" %}
+    ```text
+    helm list -n mayastor
+    ```
+    {% endtab %} 
+    {% tab title="Sample Output" %}
+    ```text
+    NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                           APP VERSION
+    mayastor        mayastor        1               2023-04-25 19:20:53.43928058 +0000 UTC  deployed        mayastor-2.1.0                  2.1.0
+    ```
+    {% endtab %}
+    {% endtabs %}
+
+    {% tabs %}
+    {% tab title="Command to check the status of pods" %}
+    ```text
+    kubectl get pods -n mayastor
+    ```
+    {% endtab %}   
+    {% tab title="Sample Output" %}
+    ```text
+    NAME                                         READY   STATUS    RESTARTS   AGE
+    mayastor-agent-core-7d7f59bbb8-nwptm         2/2     Running   0          34m
+    mayastor-agent-ha-node-fblrn                 1/1     Running   0          6m29s
+    mayastor-agent-ha-node-g6rf9                 1/1     Running   0          6m29s
+    mayastor-agent-ha-node-ktjvz                 1/1     Running   0          6m29s
+    mayastor-api-rest-6d774fbdd8-hgrxj           1/1     Running   0          34m
+    mayastor-csi-controller-6469fdf8db-bgs2h     3/3     Running   0          34m
+    mayastor-csi-node-7zm2v                      2/2     Running   0          34m
+    mayastor-csi-node-gs76x                      2/2     Running   0          34m
+    mayastor-csi-node-mfqfq                      2/2     Running   0          34m
+    mayastor-etcd-0                              1/1     Running   0          4m7s
+    mayastor-etcd-1                              1/1     Running   0          5m16s
+    mayastor-etcd-2                              1/1     Running   0          6m28s
+    mayastor-io-engine-6n6bh                     2/2     Running   0          25m
+    mayastor-io-engine-7gpsj                     2/2     Running   0          25m
+    mayastor-io-engine-95jjn                     2/2     Running   0          25m
+    mayastor-loki-0                              1/1     Running   0          34m
+    mayastor-obs-callhome-588688bb4d-w9dl4       1/1     Running   0          34m
+    mayastor-operator-diskpool-8cd67554d-c4zpz   1/1     Running   0          34m
+    mayastor-promtail-66cj6                      1/1     Running   0          34m
+    mayastor-promtail-cx9m7                      1/1     Running   0          34m
+    mayastor-promtail-t789g                      1/1     Running   0          34m
+    nats-0                                       2/2     Running   0          45m
+    nats-1                                       2/2     Running   0          45m
+    nats-2                                       2/2     Running   0          45m
+    ```
+    {% endtab %}
+    {% endtabs %}   
+
+    {% tabs %}
+    {% tab title="Command to check the application and volume" %}
```

Comment:
The command only lists volumes, though the above description says application and volume.

Reply:
Removed `application` from the text.